docs/pages/deploy-a-cluster/helm-deployments/aws.mdx

```diff
@@ -19,6 +19,19 @@ cluster to Teleport.
 
 (!docs/pages/kubernetes-access/helm/includes/teleport-cluster-prereqs.mdx!)
 
+## Choose a Kubernetes namespace and Helm release name
```

Comment:
I think we can remove this heading, since these instructions can be read as part of the Prerequisites.

Reply:
This was kinda deliberate, as I've seen that users tend to skip over text between headings and not pay attention to it. Having a separate heading with the box for edits immediately underneath increases the chance that they'll actually edit the values IMO.